### ISSUE TITLE
Webcam not working when I have IR and RCB image devices (Only chrome)

### ISSUE THREAD

**Issue by lawanyaselvamani** [mozmorris/react-webcam#200]
@mozmorris First of all thank you so much for giving this loving library. Its so cool. 
But I found one blocking issue, I update it below. Please suggest some ideas to solve them or the fix.

In Chrome webcam not working (shows a blank white screen)  and In other browsers also all seems fine.

I sensed it is somewhat due to IR image devices (https://answers.microsoft.com/en-us/windows/forum/all/hardware-access-error-when-trying-to-make-video/f86ed174-6d30-422c-869e-19e7ad6c92e2) and disabling them with device driver works fine for me.

But It is not fare right ? We couldn't able to setup each users machine to disable IR device driver.
So Could you please give any ideas or fix ?

**Comment by mozmorris** [comment]
@lawanyaselvamani thanks for this report. Tbh, I'm not sure what can be done here. This sounds like a bug in Windows.

My suggestion would be to allow the user to choose the device via a dropdown or button. But if the presence of an IR device causes an error with other video sources, I'm not sure this would fix the issue.

See https://codepen.io/mozmorris/pen/KKwdOPO

**Comment by lawanyaselvamani** [comment]
@mozmorris thanks for the reply. But I've tried with giving drop-down. But it creates trouble in Firefox, there functional collapse occurring between Firefox video source drop-down in permission prompt and the drop-down I given by code. (working well chrome, other browsers.. except Firefox). The problem is Firefox permission prompt having the video source drop-down. 
I also raised some questions regarding it in stack exchange https://security.stackexchange.com/questions/226275/customize-default-prompt-options-camera-to-share-when-i-trigger-camera-in-fire, but I couldn't able to find right solution yet.
